week6.js

```diff
@@ -0,0 +1,136 @@
+
+
+// Heyam, Week6
+
+//1.
+let arabic=("Marhabaan, Ealam!"); //Arabic language
+let turkish=("Merhaba, Dunya!"); //Turkish language
+let kurdish=("Silav, Dunay!"); //Kurdish language
+console.log(arabic);
+console.log(turkish);
+console.log(kurdish);
+
+
+
+
+//2.
+console.log('Im awesome');
+console.log('I\'m awesome');
+
+
+
+
+
+//3.
+
+let age; 
+console.log("I am not sure what the value is ");
+console.log(age);
+age=20;
+console.log("the value of the age is 20");
+console.log(age);
+
+
+
+
+
+//4
+let name="Heyam";
+console.log("the value of my string is going to be: Heyam");
+console.log(name);
+name="Riham";
+console.log("the value of my string is going to be: Riham");
+console.log(name);
+
+
+
+
+
+//5
+let number= 7.25;
+console.log(number);
+let roundedNumber= Math.round(number); //expected output 7
+console.log("Number after rounding is " + roundedNumber);
+
+//expected output 7.25
+let highestNumber=Math.max(number, roundedNumber);
+console.log(highestNumber);
+
+
+
+
+
+//6
+let myLibrary= [];
+console.log(" I think undifind");
+console.log(myLibrary);
+let myFavoriteAnimals= [ "Cat", "Birds", "Squirrel" ]; //my favorite animals.
+console.log("This array includes my favorite animals");
+console.log(myFavoriteAnimals);
+myFavoriteAnimals.push("Kitten");
+console.log(myFavoriteAnimals);
+
+
+
+
+
+//7 
+let myString = "this is a test";
+console.log(myString);
+console.log(myString.length);
+
+
+//8
+console.log(7%2) // the remainder after division should be 1
+console.log(6%2) // the remainder after division should be 0
+console.log(200%60) // the remainder after division should be 20
+
+
+
+
+
+//9
+let MultipleTypes=[2, 4,"books",true]
+
+let infinityComparison = 6/0 === 10/0;
+
+//10
+const grid = [[0, 1], [1, 1], [2, 1], [3, 1]];
+let item = grid[2][1];
+console.log(item);
+
+
+//11
+let counter = 0;
+console.log('first method');
+counter = counter + 2;
+console.log(counter);
+console.log('second method');
+counter++;
+counter++;
+console.log('Third method');
+counter +=2;
+
+
+//12
```

Comment:
It's good you used an object for this question!
Also, it's good you used strings, numbers and Booleans for the different data types.